### ISSUE TITLE
Cannot open File

### ISSUE THREAD

**Issue by Sonka-a** [OpenOrienteering/mapper#2300]
![Screenshot 2024-11-24 145326](https://github.com/user-attachments/assets/ed13f7ea-aa42-474b-9312-59f165265e36)

I get this message
I had this problem in the past, so if somebody would be so kind to explain what to do i would really appreciate it.

### Configuration
Mapper Version: 0.9.5.
Operating System: Windows 10

Thank you in advance!

**Comment by krticka** [comment]
This problem may mean that the file is not complete. For example, when copying to/from a mobile device, the file is not completely transferred. If this may be the case, I would try copying the file again.

**Comment by Sonka-a** [comment]
No, I don't think that could be it. I don't remember transferring it.
But thank you!

**Comment by dl3sdo** [comment]
Have you checked with an editor whether it's a file corruption similar to that in #2260 ?

**Comment by Sonka-a** [comment]
I didn't.
I think that it is the same problem too

**Comment by Sonka-a** [comment]
I tried to correct it in notepad as open orienteering told me to in the error messages, but it keeps getting back to line 64 and tells me to correct the mistake i just corrected.
Can I please send it to you in e-mail again?
You know it better than I do

**Comment by dl3sdo** [comment]
Yes, send it to me via email as last time and will try to repair it.
